build.sbt

```diff
@@ -7,8 +7,11 @@ lazy val backend = (project in file("backend"))
   .dependsOn(core % "test->test;compile->compile")
   .settings(backendSettings:_*)
 
+lazy val gcsfilesystem = (project in file("gcsfilesystem"))
+  .settings(gcsFileSystemSettings:_*)
+
 lazy val engine = (project in file("engine"))
-  .dependsOn(core % "test->test;compile->compile", backend)
+  .dependsOn(core % "test->test;compile->compile", gcsfilesystem % "test->test;compile->compile", backend)
```

Comment:
I don't understand that - I thought the whole point of this was that engine could do FS stuff without worrying about a backend?